File: Jewels and stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        ans = 0
        arr=[0]*256
        for i in jewels:
            arr[ord(i)]=1
        for i in stones:
            if arr[ord(i)]:
                ans+=1
        return ans

File: solutions/Jewels and stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int arr[256]={0};
        int ans=0;
        for (int i=0; i<jewels.size(); i++){
            arr[int(jewels[i])]=1;
        }
        for (int i=0; i<stones.size(); i++){
            if (arr[int(stones[i])]==1)
                ans++;
        }
        return ans;
    }
};